### ISSUE TITLE
Using display to graph testing error

### ISSUE THREAD

**Issue by kiraicode** [phillipi/pix2pix#96]
Hello,
I have been trying to use the display package to implement a **test error** curve in addition to the training error curve.
Since I am inexperienced with Torch, I am asking for help to achieve this.

Thank you!

**Comment by phillipi** [comment]
One way would be to create a test dataloader, something like `local test_data = data_loader.new(opt.nThreads, opt)', with `opt.phase=test`. `test_data:getBactch()` will then load test images. You can then run `fDx` and `fGx` on this test data to get the loss values. Finally plot the results in a similar fashion to line 399 of `train.lua`.

Be careful not to overfit the test set though! You may want to use val data instead, e.g. set `opt.phase=val`.

**Comment by kiraicode** [comment]
Thank you so much, this go inside of test.lua correct? or train.lua?

**Comment by phillipi** [comment]
If you want to see the curve during training, then you should put it in `train.lua`. If you want to just see the loss values after training, then you can put it in `test.lua`.
